### PR TITLE
Enhance `vite-plugin-monkey/src/node/userscript/index.ts`

### PR DESCRIPTION
**Pull request by Malix-Off**

### CODE REVIEW

File: packages/vite-plugin-monkey/src/node/userscript/index.ts [hunk 1]
```diff
@@ -353,31 +353,22 @@ export const finalMonkeyOptionToComment = async ({
...
-      .filter((s) => s[0] == 'resource')
-      .map((s) => {
-        if (s[1].length > maxLen) {
-          maxLen = s[1].length;
+    let maxResourceLen = 0;
+    let maxAttrLen = 0;
+    
+    attrList.forEach(([attr, value]) => {
+      if (attr === 'resource') {
+        if (value.length > maxResourceLen) {
```
Comment (lisonge) [inline packages/vite-plugin-monkey/src/node/userscript/index.ts:361]: the maxResourceLen is wrong value

**Comment by lisonge** [comment]
the Enhance of stringSort  is worthy of reference

**Comment by lisonge** [comment]
> Please open this pull request again. New changes fixes stuff

image

`Reopen` is not clickable

**Comment by Malix-Off** [comment]
Shit ok np. You can pull https://github.com/lisonge/vite-plugin-monkey/pull/62/commits/a7bf56b39015a82a6ab55e2c7c2ff1e3ab8a06da manually yourself then

**Comment by lisonge** [comment]
> Shit ok np. You can pull [a7bf56b](https://github.com/lisonge/vite-plugin-monkey/commit/a7bf56b39015a82a6ab55e2c7c2ff1e3ab8a06da) manually yourself then

now stringSort is <https://github.com/lisonge/vite-plugin-monkey/commit/df8365fbd3a6e24fa12597e9d2940e4ca8593f06>

**Pull request closed without merging**